Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -97,6 +97,54 @@ void menu_backlash();
 
 #endif
 
+#if HAS_FILAMENT_SENSOR
+
+  #define RUNOUT_EDIT_ITEMS(F) do{ \
+    EDIT_ITEM_N(bool, F, MSG_RUNOUT_MODE_NONE, &runout.enabled[F]); \
+    ACTION_ITEM_N(F, MSG_RUNOUT_MODE_NONE, []{ set_runout_mode_none(F);}); \
+    ACTION_ITEM_N(F, MSG_RUNOUT_MODE_HIGH, []{ set_runout_mode_high(F);}); \
+    ACTION_ITEM_N(F, MSG_RUNOUT_MODE_LOW, []{ set_runout_mode_low(F);}); \
+    ACTION_ITEM_N(F, MSG_RUNOUT_MODE_MOTION, []{ set_runout_mode_motion(F);}); \
```

Comment:
I'll pop in soon and implement this as a selector that shows current mode and a sub-menu to select the new mode.